config/schedule.rb

```diff
@@ -20,6 +21,8 @@
 # end
 
 # Learn more: http://github.com/javan/whenever
-every :day, at: "12.05am" do
-  rake "shedule:alerts"
+School.all.each do |school|
+  every :day, at: school.todays_admin_reminder_time do
```

Comment:
How will you make sure that cron job is updated whenever school timing are changed ?